app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] It will be good if you can implement `confirmable `options for new users; new users email should be confimed before they can sign_in to the app. ℹ️ This [link can be a quick guide](https://www.bogotobogo.com/RubyOnRails/RubyOnRails_Devise_000_Authentication_Sending_Confirmation_Email.php).